docs/targets/substrate.rst

```diff
@@ -1,7 +1,7 @@
 Parity Substrate
 ================
 
-Solang works with Parity Substrate 3.0. Note that for recent Substrate Version, cross-contract calls as well as using `address`
+Solang works with Parity Substrate 3.0. Note that for recent Substrate versions, cross-contract calls as well as using `address`
```

Comment:
Sorry but I need to hijack the topic quickly here:

Given the metadata PR gets merged and I fix events with topics (should be doable next week), we have basically all regressions fixed for `substrate-contracts-node` version 0.21 which supports `ink!` v3. Since `ink!` v4 will be released only early next year, could we have a solang release with the mentioned fixes in? Would be great because then we could say use solang v0.14 together with `substrate-contracts-node` v0.21 and you are good.

Reply:
Good idea.